pkgs/by-name/su/sublist3r/package.nix

```
@@ -0,0 +1,42 @@
+{ lib, python3Packages, fetchFromGitHub }:
+
+python3Packages.buildPythonApplication rec {
+  pname = "sublist3r";
+  version = "1.1.0";
+
+  src = fetchFromGitHub {
+    owner = "aboul3la";
+    repo = "Sublist3r";
+    rev = "refs/heads/master"; # Alternatively, use a specific commit or tag
+    sha256 = "0iz94bqsx6v4jj68h4673hq5z1ymlbcjrm8ybnb0w7q863gdpfcy";
```

SRi hashes are preferred nowadays.

```suggestion
    hash = "sha256-.....";
```